### DISCUSSION TITLE
Does it working when docker daemon is running in rootless mode ?

### DISCUSSION THREAD

**Discussion by Barnoux** [siemens/edgeshark#53] [Q&A] [answered]
Hello,

See the doc for more information about what i'm mean by rootless docker daemon  https://docs.docker.com/engine/security/rootless/

Regards,

BBA

**Comment by thediveo** [comment, marked as answer]
We've never tested this, and we don't have resources to look into it. You can either have system diagnosis or rootless lock-down.

On top of this, you'll use a completely different networking infrastructure where large parts of its state are hidden in user space processes, and these processes neither offering diagnosis nor telemetry.
